app/src/main/java/org/httpsknighthacks/knighthacksandroid/MainActivity.java

```diff
@@ -36,7 +39,18 @@
 
     public RequestQueueSingleton mRequestQueueSingleton;
 
-    private String ANNOUNCEMENTS_TOPIC = "ANNOUNCEMENTS";
+    private String GENERAL_TOPIC = "GENERAL";
```

Comment:
It's good practice to make these String members static final if they aren't going to be modified.